src/gui/member_select.ts

```diff
@@ -235,19 +231,13 @@ export class GuiMemberSelect extends GuiSubscreen {
 		MainCanvas.textAlign = "center";
 		DrawButton(1815, 75, 90, 90, "", "White", "Icons/Exit.png", "Back");
 
-		MainCanvas.beginPath();
-		MainCanvas.moveTo(125, 176);
-		MainCanvas.lineTo(1566, 176);
-		MainCanvas.stroke();
+		DrawEmptyRect(125, 176, 1441, 0, "#ddd");
 
 		DrawText(`Name`, 383, 222, "Black");
 		DrawText(`Member number`, 780, 222, "Black");
 		DrawText(`Note`, 1100, 222, "Black");
 
-		MainCanvas.beginPath();
-		MainCanvas.moveTo(125, 265);
-		MainCanvas.lineTo(1566, 265);
-		MainCanvas.stroke();
+		DrawEmptyRect(125, 265, 1441, 0, "#ddd");
```

Comment:
The default stroke color is `Black`.
```suggestion
		DrawEmptyRect(125, 265, 1441, 0, "Black");
```